src/js/components/Radar.jsx

```diff
@@ -13,12 +14,22 @@ export default class Radar extends Component {
     }).isRequired,
   };
 
-  renderRadar = () => {
+  constructor(props) {
```

Comment:
This constructor seems unnecessary, unless you really need to create the radar before componentDidMount.